recipes/cld2-cffi/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "cld2-cffi" %}
+{% set version = "0.1.4" %}
+{% set sha256sum = "3a29948364ed1e426c5bf542832eee208b1c70c0ad512b1a99bec0e6486f6c67" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win32 or linux32]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>34]
+
+requirements:
+  build:
+    - toolchain
+    - msinttypes  # [win and py<35]
+    - python
+    - setuptools
+    - cffi
+    - six
```

Comment:
Yeah, the error is informing you that `conda-build` tried to set the macOS deployment target to 10.7, but XCode complains there is no 10.7 SDK so it switches to 10.9. The `toolchain` just sets us to 10.9 so that is fine.